dataverse-webapp/src/main/webapp/dataverse_header.xhtml

```diff
@@ -214,10 +214,29 @@
                                 </ul>
                             </li>
                         </ui:fragment>
+            
+                        <div id="accessibility-controls">
+                            <h:outputText value="#{bundle['header.accessibility.contrast']}" />: 
+                            <ul id="high-contrast-mode-selector">
+                                <li>
+                                    <button class="default" title="#{bundle['header.accessibility.contrast.default']}"></button>
+                                </li>
+                                <li>
+                                    <button class="yellow-on-black" title="#{bundle['header.accessibility.contrast.yellowOnBlack']}"></button>
+                                </li>
+                            </ul>
+                        </div>
+
+                        <script type="text/javascript">
+                            // Some pages reload the header, we need to ensure the button bindings stay functional.
+                            // Example: API Token tab on user account page.
+                            accessibilityBindButtonEvents();
+                        </script>
```

Comment:
The is a problem on statistics page because it produces invalid html:
![Zrzut ekranu z 2020-04-15 13-58-48](https://user-images.githubusercontent.com/8190961/79334951-7e6de100-7f21-11ea-9380-856991fb6515.png)
As a solution we can do on metrics.xhtml:
`<f:view locale="#{dataverseSession.localeCode}">` -> `<f:view locale="#{dataverseSession.localeCode}" />`
And remove closing tag at the end oof the file: `</f:view>`

Reply:
But as we discussed this will be corrected in separate task